app/views/invite_requests/_invite_request.html.erb

```diff
@@ -6,7 +6,7 @@
 
 <!--main content-->
 <p>
-  <%= ts("You are currently number <strong>%{position}</strong> on our waiting list!", position: InviteRequest.where("position <= #{invite_request.position}").count).html_safe %>
+  <%= ts("You are currently number <strong>%{position}</strong> on our waiting list!", position: InviteRequest.where(["position <= ?", invite_request.position]).count).html_safe %>
```

Comment:
If at all possible, I'd _love_ to take the opportunity to move `InviteRequest.where(["position <= ?", invite_request.position]).count` out of the view and put it in the controller instead, maybe as `@position_in_queue` or something.